src/components/augmint-ui/header/styles.js

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+import theme from "styles/theme";
+
+const BaseHeader = `
+    color: ${theme.colors.white};
+    font-family: MaisonNeue,'Helvetica Neue',Arial,Helvetica,sans-serif;
+    font-weight: 200;
+    padding: 0;
+`;
+
+const HeaderH3 = `
+    font-size: 26px;
```

Comment:
... and `${theme.typography.fontSizes.h3};` also please convert these values to rem: `24/16 = 1.5rem`